consensus/service/src/consensus_service.rs

```diff
@@ -636,70 +646,165 @@ impl<
     /// Helper method for creating the get config json function needed by the GRPC admin service.
     fn create_get_config_json_fn(&self) -> GetConfigJsonFn {
```

Comment:
The idea here is to organize the function around getting the ledger status (highest_block, block_signature) and getting the sync status (is_behind, highest_peer_block_index).